server/src/core/python_arch_builder.rs

```diff
@@ -83,6 +84,11 @@ impl PythonArchBuilder {
                 SyncOdoo::is_in_workspace_or_entry(session, path.as_str());
             self.file.borrow_mut().set_in_workspace(in_workspace);
         }
+        if symbol.borrow().typ() == SymType::PACKAGE(crate::constants::PackageType::MODULE) {
```

Comment:
```suggestion
        if symbol.borrow().typ() == SymType::PACKAGE(PackageType::MODULE) {
```
Consistency suggestion: and add it in the imports, as it is done with SymType in the import